dagfactory/dagbuilder.py

```diff
@@ -10,6 +10,7 @@
 from airflow.contrib.operators.kubernetes_pod_operator import KubernetesPodOperator
 from airflow.models import BaseOperator
 from airflow.operators.python_operator import PythonOperator, BranchPythonOperator
+from airflow.sensors.python import PythonSensor
```

Comment:
Can you move this to line 42? `PythonSensor` wasn't introduced until later and I'd like to maintain backwards compatibility with older Airflow versions if possible

Reply:
Yes sure, sorry for number of tries...

Reply:
Another question with a similar background, I face the callable Problem with the google pubsub sensor as well, the sensor has a python class for minimal processing of the messages as default or you can call an own function, both does not work out of the box, do you have an idea how to solve that?